### PR TITLE
Dco

### PR DESCRIPTION
**Pull request by nibalizer**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -83,3 +83,9 @@ To inspect the zuul VM::
 To tear down the entire stack when you're done::
 
    $ vagrant destroy
+
+
+Contributing
+============
+
+See CONTRIBUTING.md in this repository
```
Comment (eggshell) [inline README.rst]: Any reason not to include a relative link?

**Review by eggshell — APPROVED** [review]
Other than the nit about the contribution link, lgtm.

File: test/signed-off-by-test.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+success="true"
+
+for commit in $(git cherry master | cut -d " " -f 2)
+do
+    if $(git show $commit | grep -q Signed-off-by ); then
```
Comment (nibalizer) [inline test/signed-off-by-test.sh]: if we add -s to git show here, it wont emit the diff, which removes a bug where 'signed-off-by' can be in the diff.

**Comment by omgjlk** [comment]
recheck

**Comment by anne-bonny** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+	Signed-off-by: Random J Developer <random@developer.example.org>
+
+using your real name (sorry, no pseudonyms or anonymous contributions.)
+
+
+Git
+---
+
+You can do this easily in git but using ``-s`` when you run ``git commit``
```
Comment (ConsoleCatzirl) [inline CONTRIBUTING.md]: Typo: s/but/by/

File: README.rst (hunk 1 above)
Comment (ConsoleCatzirl) [reply README.rst]: Yeah, a relative link would be good

File: test/signed-off-by-test.sh [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+success="true"
+
+for commit in $(git cherry master | cut -d " " -f 2)
+do
+    if $(git show -s $commit | grep -q 'Signed-off-by:' ); then
+        :
```
Comment (ConsoleCatzirl) [inline test/signed-off-by-test.sh]: Why not negate the if instead of having an empty section here?

**Review by ConsoleCatzirl — CHANGES_REQUESTED** [review]

**Comment by anne-bonny** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by omgjlk** [comment]
recheck

**Comment by anne-bonny** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by omgjlk** [comment]
recheck

**Comment by anne-bonny** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by anne-bonny** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Review by ConsoleCatzirl — APPROVED** [review]

**Pull request merged**